### PR TITLE
Rewording parts of the README.md file

### PR DESCRIPTION
**Pull request by ewolinetz**
@bparees @mfojtik PTAL

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,14 +81,14 @@ $ curl 127.0.0.1:8080
 
 Test
 ---------------------
-This repository also provides [STI](https://github.com/openshift/source-to-image) test framework,
-which launches tests to check functionality of a simple perl application built on top of sti-perl image.
+This repository also provides a [STI](https://github.com/openshift/source-to-image) test framework,
```
Comment (bparees) [inline README.md]: S2I

File: README.md [hunk 2]
```diff
@@ -145,32 +145,32 @@ Repository organization
 
     * **`contrib/`**
 
-        This folder contains file with commonly used modules.
+        This folder contains a file with commonly used modules.
 
     * **`test/`**
 
-        This folder is containing [STI](https://github.com/openshift/source-to-image)
+        This folder contains the [STI](https://github.com/openshift/source-to-image)
```
Comment (bparees) [inline README.md]: S2I

**Comment by bparees** [comment]
2 nits and lgtm.

**Comment by ewolinetz** [comment]
addressed nits and additional appropriate STI->S2I updates

**Comment by bparees** [comment]
lgtm.

**Pull request merged**